Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.cs

```diff
@@ -189,12 +189,12 @@ private void MinThumb_KeyDown(object sender, KeyRoutedEventArgs e)
             {
                 case VirtualKey.Left:
                     RangeMin -= StepFrequency;
-                    SyncThumbs();
+                    SyncThumbs(fromMinKeyDown: true);
```

Comment:
Can we just call DragThumb after the calls to SyncThumbs?  Will that still work?  

Then we wouldn't have to pass down all the ternary logic.  And then we can just do the Tooltip Update after each of these Switch statements or in the DragThumb method maybe?

Reply:
I don't really understand what you mean, but calling DragThumb is exactly what I'm doing at the end of SyncThumbs, as you can see [https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/2849/files/c9d701b5ef7c2b0b3f812d26849b4c7e690adbb3#diff-3b81f5ca9b63aec6a53a54cebd02b969R654](here).

Reply:
I was saying rather than modifying SyncThumbs, to just make the calls to DragThumb in this KeyDown call.  i.e. do the logic here for each case rather than passing flags and consolidating all the logic into the single complex statement.